bokeh/document.py

```diff
@@ -36,6 +37,42 @@ def __init__(self, document, model):
         super(RootRemovedEvent, self).__init__(document)
         self.model = model
 
+class PeriodicCallbackAdded(DocumentChangedEvent):
+    def __init__(self, document, callback):
+        super(PeriodicCallbackAdded, self).__init__(document)
+        self.callback = callback
+
+class PeriodicCallbackRemoved(DocumentChangedEvent):
+    def __init__(self, document, callback):
+        super(PeriodicCallbackRemoved, self).__init__(document)
+        self.callback = callback
+
+class PeriodicCallback(object):
+    def __init__(self, document, callback, period, id=None):
+        if id is None:
+            self._id = str(uuid.uuid4())
+        else:
+            self._id = id
+
+        self._document = document
+        self._period = period
+        self._callback = callback
+
+    @property
+    def period(self):
```

Comment:
whenever I have a duration like this I find it helpful to include the units in the name, like `period_milliseconds` (I don't know it's milliseconds! just an example). But if not at least document it.